<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientInput</class>
 <widget class="QWidget" name="ClientInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>241</width>
     <height>120</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>M.</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mme.</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Melle.</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="comboBox_2">
      <item>
       <property name="text">
        <string>M.</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mme.</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Melle.</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="comboBox_3">
      <item>
       <property name="text">
        <string>M.</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mme.</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Melle.</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="comboBox_4">
      <item>
       <property name="text">
        <string>M.</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mme.</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Melle.</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
